teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative './person'
+
+class Teacher < Person
+  def initialize(specialization, age, name)
+    super(age, name)
+    @specialization = specialization
+  end
+
+  def play_hooky
+    '¯\(ツ)/¯'
+  end
```

Comment:
- [ ] This method should belong to the `Student` class instead of the `Teacher` class. Kindly swap it with the `can_use_services?` you currently have in the `Student` class

Reply:
Done!, thanks